resources/custom.Makefile

```diff
@@ -17,6 +31,41 @@ $(TMP_DATA)/phenogeno_%.zip:
 	mkdir -p $(TMP_DATA)
 	wget https://data.monarchinitiative.org/exomiser/data/$*.zip -O $@
 
+.PHONY: download-exomiser
+download-exomiser: $(addprefix $(RUNNERS_DIR)/exomiser-,$(EXOMISER_VERSIONS))
+
+$(TMP_DATA)/exomiser-cli-%-distribution.zip:
+	mkdir -p $(TMP_DATA)
+	wget https://github.com/exomiser/Exomiser/releases/download/$*/exomiser-cli-$*-distribution.zip -O $@
+
+$(RUNNERS_DIR)/exomiser-%: $(TMP_DATA)/exomiser-cli-%-distribution.zip
+	mkdir -p $(RUNNERS_DIR)
+	unzip $< -d $(RUNNERS_DIR)
+	mv $(RUNNERS_DIR)/exomiser-cli-$* $(RUNNERS_DIR)/exomiser-$*
+	cp $(RUNNERS_DIR)/configurations/preset-exome-analysis.yml $(RUNNERS_DIR)/exomiser-$*/
+
+
+.PHONY: download-phen2gen
+download-phen2gen: $(RUNNERS_DIR)/Phen2Gene
+$(RUNNERS_DIR)/Phen2Gene:
+	mkdir -p $(RUNNERS_DIR)
+	cd $(RUNNERS_DIR)
+	git clone https://github.com/WGLab/Phen2Gene.git $@
+	yes "$@" | bash $@/setup.sh
+
+
+.PHONY: download-gado
+download-gado: $(RUNNERS_DIR)/gado
+$(RUNNERS_DIR)/gado:
```

Comment:
For transparency, I personally prefer it if a goal is `.PHONY` if the file that corresponds to the name is not actually a file, but in this case, its fine!